golemcli/src/commands/status.rs

```diff
@@ -0,0 +1,603 @@
+    fn to_json(&self) -> Result<Value, Error> {
+        Ok(serde_json::to_value(&self)?)
+    }
+
+    fn print(&self) -> Result<(), Error> {
+        let mut section_builder = SectionBuilder::new( "  ");
+
+        section_builder
+            .new_section("General")
+            .entry(
+                "Process State",
+                &self.running_status.process_state.to_string(),
+            )
+            .new_subsection("Components Status")
+            .entry(
+                "docker",
+                self.running_status
+                    .component_statuses
+                    .docker_status
+                    .as_ref()
+                    .unwrap_or(&String::from("unknown")),
+            )
+            .entry(
+                "hyperdrive",
+                self.running_status
+                    .component_statuses
+                    .hyperdrive
+                    .as_ref()
+                    .unwrap_or(&String::from("unknown")),
+            )
+            .entry(
+                "client",
+                self.running_status
+                    .component_statuses
+                    .client
+                    .as_ref()
+                    .unwrap_or(&String::from("unknown")),
+            );
+
+        if self.running_status.component_statuses.hypervisor.is_some() {
+            section_builder.entry(
+                "hypervisor",
+                self.running_status
+                    .component_statuses
+                    .hypervisor
+                    .as_ref()
+                    .unwrap(),
+            );
+        }
+        let connection_status = if self.net_status.is_connected {
+            String::from("ONLINE")
+        } else {
+            String::from("OFFLINE")
+        };
+
+        section_builder
+            .end_section()
+            .entry("Golem version", &self.running_status.golem_version)
+            .entry("Node name", &self.running_status.node_name)
+            .entry("Network", &self.running_status.network.to_string())
+            .new_subsection("Disk usage")
+            .entry(
+                "Received files",
+                &self.running_status.disk_usage.received_files,
+            )
+            .entry(
+                "Distributed files",
+                &self.running_status.disk_usage.distributed_files,
+            )
+            .end_section()
+            .end_section()
+            .new_section("Network")
+            .entry("Connection", &connection_status)
+            .entry(
+                "Port statuses",
+                &format!("{:?}", self.net_status.port_status),
+            )
+            .entry("Nodes online", &self.net_status.nodes_online.to_string())
+            .end_section()
+            .new_section("Account")
+            .entry("ETH address", &self.account_status.eth_address)
+            .entry("GNT available", &self.account_status.gnt_available)
+            .entry("ETH available", &self.account_status.eth_available)
```

Comment:
```
  GNT available: 990.422335685398871004 GNT
  ETH available: 18.767548838 ETH
```
pls round this, maybe to 3 or 5 decimals

```
  GNT available: 990.422 GNT
  ETH available: 18.76754 ETH
```

Reply:
It is not good idea to round this? What if someone has 0.000001 GNT?